package.json

```diff
@@ -4,19 +4,20 @@
   "description": "Timesheet app using the SAP Cloud SDK and Express.js",
   "main": "src/index.ts",
   "scripts": {
+    "megacleaninstall": "rm -f package-lock.json && find . -name \"node_modules\" -exec rm -rf '{}' + && find db/src -name \"gen\" -exec rm -rf '{}' + && find . -name \"sqlite.db\" -exec rm -rf '{}' + && npm i",
     "init": "npm i && node initialize.js",
     "preinstall": "rm -rf node_modules/@sap/cds-services",
     "cds-build": "npx cds build/all --clean",
-    "cds-deploy:local": "npx cds deploy",
-    "cds-deploy:hana": "npx cds deploy --to hana:timesheet-hdi-container-codejam-<participantId>",
+    "cds-deploy:local": "npx cds deploy --to sqlite",
```

Comment:
I guess `cds-deploy` would be a better name, given, that there is no difference between local and prod anymore.